### DISCUSSION TITLE
Sort open tickets widget by latest tickets

### DISCUSSION THREAD

**Discussion by acoolov** [YetiForceCompany/YetiForceCRM#15306] [General] [answered]
Is it possible for the dashboard widget to always show open tickets that are the latest ticket number? Or maybe last modified tickets? The problem is, the widget right now is showing open tickets but as they accumulate, it keeps showing only the old open tickets and the latest ones are not shown. Would be amazing if it always updated and showed latest tickets worked on.

**Comment by rskrzypczak** [comment, marked as answer]
Yes, it is quite simple.
Create the filter you want, add sorting
![image](https://user-images.githubusercontent.com/10221999/137068162-ac36ffb6-2bed-4dfd-8caa-ee741d73a2df.png)
and then add the widget.
![image](https://user-images.githubusercontent.com/10221999/137068367-f9c3745a-7525-4de9-9e11-b05ce38dee0d.png)

**Comment by acoolov** [comment]
Very helpful, thanks!
